### PR TITLE
pkg/pty: modify so it compiles under Plan 9

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

File: cmds/exp/console/console.go [hunk 1]
```diff
@@ -47,27 +47,7 @@ func main() {
...
-		u, err := openi8042()
-		if err != nil {
-			log.Fatalf("Console exits: sorry, can't get an i8042: %v", err)
-		}
-		in, out = u, os.Stdout
-	case *serial == "stdio":
-	default:
-		log.Fatalf("Console exits: console must be one of stdio, i8042, or an IO port with a leading 0 (e.g. 0x3f8)")
-	}
+	in, out, err := console(*serial)
```
Comment (hugelgupf) [inline cmds/exp/console/console.go:50]: This is an ineffectual assign. err is reassigned immediately by p.Start, so err is unused in this assignment.

Comment (rminnich) [reply cmds/exp/console/console.go:50]: yeah.

Comment (rminnich) [reply cmds/exp/console/console.go:50]: fixed.

File: cmds/exp/console/console_linux.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
```
Comment (sevki) [inline cmds/exp/console/console_linux.go:1]: nit
```suggestion
// Copyright 2015-2020 the u-root Authors. All rights reserved
```

File: cmds/exp/console/console_plan9.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
```
Comment (sevki) [inline cmds/exp/console/console_plan9.go:1]: nit
```suggestion
// Copyright 2015-2020 the u-root Authors. All rights reserved
```

File: pkg/pty/pty_plan9.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
```
Comment (sevki) [inline pkg/pty/pty_plan9.go:1]: nit
```suggestion
// Copyright 2015-2020 the u-root Authors. All rights reserved
```

**Pull request merged**